app/lib/test_applications.rb

```diff
@@ -116,7 +116,7 @@ def create_application_to_courses(
       previous_application_form = nil
       candidate = candidate.presence || FactoryBot.create(
         :candidate,
-        email_address: "#{first_name.downcase}.#{last_name.downcase}@example.com",
+        email_address: "#{first_name.downcase}.#{last_name.downcase}#{rand(10000)}@example.com",
```

Comment:
Is it not best to use [sequences](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences) for this?

Reply:
Good point. It doesnt go into as much detail in the documentation but as this is not the intention of the gem it probably won't work in this scenario.